src/DataBox/DataBox.Test/DataBox.Test.csproj

```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Azure.Management.DataBox" Version="1.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This is already done here: https://github.com/Azure/azure-powershell/blob/master/src/Az.Test.props#L38

Reply:
Unable to understand what is already done here. Can you please clarify in detail?

Reply:
Please look at the file I linked - you are importing it (as all test projects do) and it already copies all of your sessionrecords.  This added itemgroup in your csproj file is redundant.  Remove it, please.

Reply:
Thanks for clarification. Removed the redundant entry in csproj file.